### ISSUE TITLE
autoAtmosChange says API key isn't valid

### ISSUE THREAD

**Issue by clikuki** [multimokia/MAS-Submod-Auto-Atmos-Change#10]
I'm using Doki Doki Mod Manager and I'm on Linux. I put my API key in `/home/clikuki/.config/DokiDokiModManager/GameData/installs/monika-after-story/install/apikey.txt` and I've already checked if my API key was valid, but it still says "Please add a valid API key to use this submod".

**Comment by clikuki** [comment]
## Adding additional pictures
My DDLC folder
![fileDirectory](https://user-images.githubusercontent.com/73450123/120146472-bdd7b980-c217-11eb-85ad-c91b1a39049e.png)   

My submods
![installedSubmods](https://user-images.githubusercontent.com/73450123/120146818-40607900-c218-11eb-8b27-90735d7772d1.png)

AutoAtmos Submod saying API key isn't valid
![subMods](https://user-images.githubusercontent.com/73450123/120146522-cc25d580-c217-11eb-9658-3e637fce3a9b.png)

Testing my API key on firefox
![testApiKey](https://user-images.githubusercontent.com/73450123/120147245-ef9d5000-c218-11eb-8a01-5ee0b22650e4.png)

**Comment by multimokia** [comment]
I still need to look into OpenWeatherMap, it's likely that new keys have been depreciated for the legacy api (Python 2 endpoint)

If that's the case it is possible that new keys will not work with Auto Atmos Change, and we'll likely be stuck with this until Renpy 8 releases and MAS is able to update its renpy version to that.

**Comment by Sirix1995** [comment]
Same here. I generated a key, but it don't work. Here's the traceback.

I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 1508, in script call
  File "game/event-handler.rpy", line 3114, in script call
  File "game/Submods/Auto Weather Change/awc_topics.rpy", line 81, in script
    $ awc_savePlayerCityCountryTup((temp_city, awc_getCityCountry(temp_city)))
  File "game/Submods/Auto Weather Change/awc_topics.rpy", line 81, in <module>
    $ awc_savePlayerCityCountryTup((temp_city, awc_getCityCountry(temp_city)))
  File "game/Submods/Auto Weather Change/Utilities/awc_utils.rpy", line 252, in awc_getCityCountry
    return store.awc_globals.owm.weather_at_place(city_name).get_location().get_country()
UnauthorizedError: Your API subscription level does not allow to perform this operation
Reason: Invalid API key

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1508, in script call
  File "game/event-handler.rpy", line 3114, in script call
  File "game/Submods/Auto Weather Change/awc_topics.rpy", line 81, in script
    $ awc_savePlayerCityCountryTup((temp_city, awc_getCityCountry(temp_city)))
  File "D:\After\Doki Doki Literature Club\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\After\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/Submods/Auto Weather Change/awc_topics.rpy", line 81, in <module>
    $ awc_savePlayerCityCountryTup((temp_city, awc_getCityCountry(temp_city)))
  File "game/Submods/Auto Weather Change/Utilities/awc_utils.rpy", line 252, in awc_getCityCountry
    return store.awc_globals.owm.weather_at_place(city_name).get_location().get_country()
  File "build/bdist.linux-x86_64/egg/pyowm/webapi25/owm25.py", line 231, in weather_at_place
    'lang': self._language})
  File "build/bdist.linux-x86_64/egg/pyowm/commons/weather_client.py", line 94, in call_API
    return self._lookup_cache_or_invoke_API(self._cache, url, timeout)
  File "build/bdist.linux-x86_64/egg/pyowm/commons/weather_client.py", line 58, in _lookup_cache_or_invoke_API
    raise unauthorized_error.UnauthorizedError('Invalid API key')
UnauthorizedError: Your API subscription level does not allow to perform this operation
Reason: Invalid API key

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.8.3-unstable.2022.03.08.158

**Comment by Oliver-0001** [comment]
> Same here. I generated a key, but it don't work. Here's the traceback.
> 
> I'm sorry, but an uncaught exception occurred.
> 
> While running game code: File "game/script-ch30.rpy", line 1508, in script call File "game/event-handler.rpy", line 3114, in script call File "game/Submods/Auto Weather Change/awc_topics.rpy", line 81, in script $ awc_savePlayerCityCountryTup((temp_city, awc_getCityCountry(temp_city))) File "game/Submods/Auto Weather Change/awc_topics.rpy", line 81, in $ awc_savePlayerCityCountryTup((temp_city, awc_getCityCountry(temp_city))) File "game/Submods/Auto Weather Change/Utilities/awc_utils.rpy", line 252, in awc_getCityCountry return store.awc_globals.owm.weather_at_place(city_name).get_location().get_country() UnauthorizedError: Your API subscription level does not allow to perform this operation Reason: Invalid API key
> 
> -- Full Traceback ------------------------------------------------------------
> 
> Full traceback: File "game/script-ch30.rpy", line 1508, in script call File "game/event-handler.rpy", line 3114, in script call File "game/Submods/Auto Weather Change/awc_topics.rpy", line 81, in script $ awc_savePlayerCityCountryTup((temp_city, awc_getCityCountry(temp_city))) File "D:\After\Doki Doki Literature Club\renpy\ast.py", line 814, in execute renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store) File "D:\After\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode exec bytecode in globals, locals File "game/Submods/Auto Weather Change/awc_topics.rpy", line 81, in $ awc_savePlayerCityCountryTup((temp_city, awc_getCityCountry(temp_city))) File "game/Submods/Auto Weather Change/Utilities/awc_utils.rpy", line 252, in awc_getCityCountry return store.awc_globals.owm.weather_at_place(city_name).get_location().get_country() File "build/bdist.linux-x86_64/egg/pyowm/webapi25/owm25.py", line 231, in weather_at_place 'lang': self._language}) File "build/bdist.linux-x86_64/egg/pyowm/commons/weather_client.py", line 94, in call_API return self._lookup_cache_or_invoke_API(self._cache, url, timeout) File "build/bdist.linux-x86_64/egg/pyowm/commons/weather_client.py", line 58, in _lookup_cache_or_invoke_API raise unauthorized_error.UnauthorizedError('Invalid API key') UnauthorizedError: Your API subscription level does not allow to perform this operation Reason: Invalid API key
> 
> Windows-8-6.2.9200 Ren'Py 6.99.12.4.2187 Monika After Story 0.12.8.3-unstable.2022.03.08.158

Yes, i have same exact error, saying my api key subscription is not enough. but atleast my api-key is valid...
